lib/features/search/presentation/ui/widgets/search_bottom_drawer.dart

```diff
@@ -0,0 +1,234 @@
+              padding: const EdgeInsets.all(10.0),
+              child: Text(
+                "No places searched yet...",
+                style: TextStyle(
+                  fontSize: 20,
+                  fontWeight: FontWeight.bold,
+                ),
+              ));
+        }
+      },
+    );
+  }
+}
+
+class PlaceCard extends StatelessWidget {
+  final File placePhoto;
+  final PlaceEntity placeEntity;
+
+  const PlaceCard({Key key, this.placePhoto, this.placeEntity})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final navScreenBloc = BlocProvider.of<NavigationScreenBloc>(context);
+    final photoBytes = placePhoto.readAsBytesSync();
+
+    return Row(
+      children: <Widget>[
+        Container(
+          padding: EdgeInsets.only(top: 10.0, right: 10.0),
+          height: 140,
+          width: 350,
+          child: Card(
+            elevation: 5,
+            child: Row(
+              children: <Widget>[
+                Container(
+                    child: Padding(
+                  padding: const EdgeInsets.all(8.0),
+                  child: Image.memory(
+                    photoBytes,
+                    width: 140,
+                    height: 140,
+                    fit: BoxFit.fill,
+                  ),
+                )),
+                Expanded(
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: <Widget>[
+                      Row(
+                        children: <Widget>[
+                          Expanded(
+                            child: Container(
+                              margin: const EdgeInsets.only(
+                                top: 12.0,
+                                left: 12.0,
+                              ),
+                              child: Text(
+                                placeEntity.name,
+                                style: TextStyle(
+                                  fontSize: 18.0,
+                                  fontWeight: FontWeight.bold,
+                                ),
+                              ),
+                            ),
+                          ),
+                        ],
+                      ),
+                      Container(
+                        margin: const EdgeInsets.only(
+                          top: 12.0,
+                          left: 12.0,
+                        ),
+                        child: Text(
+                          placeEntity.address,
+                          style: TextStyle(
+                            fontSize: 14.0,
+                          ),
+                          maxLines: 3,
+                          overflow: TextOverflow.ellipsis,
+                        ),
+                      ),
+                    ],
+                  ),
+                ),
+              ],
+            ),
+          ),
+        ),
+        Column(
```

Comment:
Wrap this in a container and give it a fixed width/height? Whatever dimension are appropriate.

Reply:
Done